actfw_raspberrypi/capture.py

```diff
@@ -25,6 +26,14 @@ def __init__(
             camera (:class:`~picamera.PiCamera`): picamera object
 
         """
+        if is_buster():
+            warnings.warn(
+                "PiCameraCapture do not work in bullseye and PiCameraCapture will be deprecated soon.",
+                PendingDeprecationWarning,
+            )
+        elif is_bullseye():
```

Comment:
This code may call systemcall twice. However, it's very minor problem since this is expected to be called once at most. It's ok.